crates/matrix-sdk-common/src/deserialized_responses.rs

```diff
@@ -669,6 +669,23 @@ fn unknown_utd_reason() -> UnableToDecryptReason {
     UnableToDecryptReason::Unknown
 }
 
+/// A room key might be missing because the sender refused to share it or was
+/// not technically able to share it.
+/// In the protocol this is reflected by clients sending `m.room_key.withheld`
+/// to the participant that won't receive the key.
+/// We only want here to consider the subset of codes that clients should
+/// use to display specific UX.
```

Comment:
as [before](https://github.com/matrix-org/matrix-rust-sdk/pull/4275#discussion_r1860871953), the formatting here is... odd.

If you want a paragraph break, you need a blank line. If you *don't* want a paragraph break, then don't pretend that there is one.

Reply:
Can we have a formatting rule that forbid new lines in comments then? if it makes no sense

Reply:
new lines are perfectly valid, they just don't mean what you seem to think they mean.

Reply:
A related point here, if you put the whole comment on the same line `rustfmt` will happily format this for you correctly, unless of course you do wish paragraph breaks, but then you can put individual paragraphs on a single line and `rustfmt` will help you with each individual paragraph.

Reply:
> new lines are perfectly valid, they just don't mean what you seem to think they mean.

I put them for clarity when reading the source code because of the maximum length, there is no such constraint in the doc

Reply:
Not exactly sure which constraint you mean, but in case you mean a max comment length, we have this defined: https://github.com/matrix-org/matrix-rust-sdk/blob/e1f0037fd5eb3b263a471a0c515daf6bf95cd784/.rustfmt.toml#L2

So no need to think about the max comment length yourself, `rustfmt` will format comments for you.